src/components/colorbar/draw.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 var d3 = require('d3');
+var tc = require('tinycolor2');
```

Comment:
Ahhh whoops - there's a few places with `var tc = tinycolor('rgb(1,2,3)');` that I was head-referencing.